.github/workflows/docker_publish.yml

```diff
@@ -0,0 +1,38 @@
+name: Build and Publish Docker
+
+on:
+  push:
```

Comment:
limit to main only?

Reply:
It pushes images tagged with the SHA for all pushes and only releases push "latest"-tag.